CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.0] - 27-01-21
+### Changed
+- Upgrade canonical to v0.5 (#371)
+
```

Comment:
We should probably mentioned jubjub and bls as well

Reply:
Fixed